pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -672,6 +686,18 @@ func getVolumesFromPodDir(podDir string) ([]podVolume, error) {
 			}
 		}
 	}
+	mps, err := mounter.List()
```

Comment:
This is a very expensive operation to add to the primary volume manager control loop

Reply:
In here, this mounter.List() is only called once after reconciler starts.

Reply:
Great

Reply:
Ack.